src/utils/save_game/save_game.cpp

```diff
@@ -0,0 +1,46 @@
+#include "save_game.h"
+#include "../utils.h"
+#include "../../creature/creature.h"
+#include "../../creature/main_character/main_character.h"
+#include <iostream>
+#include <fstream>
+
+void SaveGame(MainCharacter& main_characher)
+{
+	std::ofstream outputFile;
+
+	outputFile.open("settings.txt");
+
+	if (outputFile.is_open())
+	{
+		outputFile << "@$%READ" << std::endl;
```

Comment:
Could you please explain this line here? Why is it like so, and how is/will it be used?

Reply:
The idea was the same with the use of @@@ , when we read @#$READ, or just the @$% we know that what follows is code  we can use for the values of HP,AC or the text size, do you have something else to recomend on that?

Reply:
This file should be just ours to modify. Therefore, there is no need for such a thing here. Simply arraying the values we want should suffice.

Reply:
I mean, it's not like the chapter files that could be modified by players (DMs) in the future. A savefile should have a pre-defined structure, without any extras in it. It's not a case of "text plus commands plus more text plus more commands with arguments plus...", like it is with chapter files.